src/lib/cli.js

```diff
@@ -28,6 +28,7 @@ program
   .option('-d, --devMode', 'Shortcut for running cucumber-js directly')
   .option('--fail-fast', 'abort the run on first failure')
   .option('--strict', 'Fail fast if a step is undefined')
+  .option('--mergedLog', 'Path, relative to logdir, merged log. Specify "" to not generate one.')
```

Comment:
There's already inconsistency in here but I think we want cmd line params hyphenated, not camelcase. Would it be more natural for paths in command line params to be relative to cwd?